### PR TITLE
SAP 2 tier user disk MD

### PR DESCRIPTION
**Pull request by MSSedusch**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* add SAP 2 tier user disk MD

### CODE REVIEW

File: sap-2-tier-user-disk-md/azuredeploy.parameters.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "sapSystemId": {
+      "value": "DEV"
+    },
+    "osType": {
+      "value": "Linux"
+    },
+    "userDiskId": {
```
Comment (bmoore-msft) [inline sap-2-tier-user-disk-md/azuredeploy.parameters.json]: is this a value that anyone can use?

File: sap-2-tier-user-disk-md/metadata.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "SAP NetWeaver 2-tier (custom managed disk)",
+  "description": "This template allows you to deploy a VM using a custom managed OS disk.",
+  "summary": "This template takes a minimum amount of parameters and deploys a VM that is customized for use with SAP NetWeaver, using a spezialized i.e. not generalized managed OS disk.",
```
Comment (bmoore-msft) [inline sap-2-tier-user-disk-md/metadata.json]: typo - specialized

Comment (MSSedusch) [inline sap-2-tier-user-disk-md/metadata.json]: fixed

File: sap-2-tier-user-disk-md/azuredeploy.parameters.json (hunk 1 above)
Comment (MSSedusch) [inline sap-2-tier-user-disk-md/azuredeploy.parameters.json]: no, removed

**Pull request merged**